v2/stacks/README.md

```diff
@@ -176,31 +171,18 @@ Unless stated below, for each stack listed, the following is assumed:
    make clean
    ```
 
-### Homepage web
-
-A stack for the home page and census hub in Web mode.
-
-The first time that you run it, you will need to generate the assets and have the zebedee content. It also assumes that you have defined `zebedee_root` env var in your system.
-
-Stack root folder is [`homepage-web`](./homepage-web/).
-See the [general guidance, above](#general-guidance-for-each-stack), first, then:
-
-1. Open your browser and check you can see the home page: `localhost:24400`
-
-1. Check you can see the census hub page: `localhost:24400/census`
+See [Using `make`](#using-make) for additional make targets and other information on using `make` to work with the stacks.
 
 ### Homepage publishing
 
 Deploys a stack for the home page in Publishing mode.
 
-The first time that you run it, you will need to generate the assets and have the zebedee content. It also assumes that you have defined `zebedee_root` env var in your system.
-
 Stack root folder is [`homepage-publishing`](./homepage-publishing/).
 See the [general guidance, above](#general-guidance-for-each-stack), first, then:
 
 1. Open your browser and check you can see the florence website: `http://localhost:8081/florence`
 
-1. Log in to florence and perform the publish journey
+2. Log in to florence and perform the publish journey
 
```

Comment:
Given you've created a proper readme in the homepage stack, should this still be here?

Reply:
Same for others.

Reply:
Have added readmes to the stacks that were missing them and have given this README a first pass. Still needs more tidying up, but decided to draw a line for now.